src/figuresClasses.ts

```diff
@@ -1,19 +1,77 @@
-export interface Figure {
 
+type Color = 'red' | 'green' | 'blue';
+
+export interface Figure {
+  shape: string;
```

Comment:
create enum for Shape

```suggestion
  shape: Shape;
```

Reply:
not fixed